types/node/v16/globals.d.ts

```diff
@@ -56,13 +56,16 @@ interface AbortController {
     abort(reason?: any): void;
 }
 
+import { EventTarget as _EventTarget } from "node:events";
```

Comment:
> Module '"node:events"' has no exported member 'EventTarget'.ts(2305)

Reply:
Yeah, this is wrong. Should this be done in a way similar to how it's done in node 18 and LTS types?

Reply:
I think yes